plugins/basic/nexus-timeline-plugin/src/main/java/org/sonatype/nexus/timeline/feeds/subscribers/ItemSubscriber.java

```diff
@@ -117,7 +117,7 @@ else if (ievt instanceof RepositoryItemValidationEventFailedFileType) {
     }
   }
 
-  private void inspectForNexus(Event<?> evt) {
+  private void inspectForNexus(Object evt) {
```

Comment:
I'd prefer making this signature `RepositoryItemEvent evt` since that's the only thing it accepts.